src/e3/testsuite/driver/adacore.py

```diff
@@ -128,14 +128,16 @@ def get_script_command_line(self) -> List[str]:
             )
 
         _, ext = os.path.splitext(self.script_file)
+
+        is_cmd = ext.startswith(".cmd")
+        must_convert = is_cmd and (
+            self.env.host.os.name != "windows" or "FORCE_SH" in self.env.discs
+        )
+
         if ext == ".py":
             return [sys.executable, self.script_file]
 
-        elif (
-            ext == ".sh"
-            or self.env.host.os.name != "windows"
-            or "FORCE_SH" in self.env.discs
-        ):
+        elif (not is_cmd) or must_convert:
```

Comment:
Extra parens not necessary (unary operators have higher precedence than binary ops)

Reply:
IMO the parens make the result of the expression more obvious. But I don't feel strongly about it, so I'll change it :)